packages/pwa/app/constants.js

```diff
@@ -41,6 +41,11 @@ export const cssColorGroups = {
     miscellaneous: 'linear-gradient(to right, orange , yellow, green, cyan, blue, violet)'
 }
 
+// Color to use for the UI surrounding the page in browsers.
+export const THEME_COLOR = '#0176D3'
+// On iOS, the title for the launch icon.
+export const APPLE_MOBILE_WEB_APP_TITLE = 'PWA Kit | Retail React App'
```

Comment:
Umm should this one not be programmatic? Btw, @kevinxh also added `sitename` to this fiile.

Reply:
https://github.com/SalesforceCommerceCloud/pwa-kit/pull/287/files#diff-ab367c48371fe0de62205294eec74c1c9d6536ce438a2c6745c5665e01434328R72